.gitignore

```diff
@@ -0,0 +1,4 @@
+.Rproj.user
```

Comment:
Do not include your gitignore in the repository, it is specific to you, you can add '.gitignore' to your '.gitignore' file